orderer/common/localconfig/config_test.go

```diff
@@ -19,7 +19,7 @@ import (
 func TestLoadGoodConfig(t *testing.T) {
 	cleanup := configtest.SetDevFabricConfigPath(t)
 	defer cleanup()
-	cfg, err := Load()
```

Comment:
All of the changes in here could be avoided by using a package flag updated by the test. While I am not a fan of package state in general, it has very limited scope here as it's a zero value that is only modified in test.

Reply:
done